scripts/aem.js

```diff
@@ -373,6 +373,53 @@ function decorateHorizontalRules(element) {
   });
 }
 
+/**
+ * Converts '<layout>' text nodes into containers to allow for nesting of blocks within layouts
+ * @param {Element} element container element
+ */
+function decorateLayouts(element) {
+  // find all '<layout>' text nodes regardless of layout type
```

Comment:
I am going to send you a Slack message about the use of Angle brackets, let's talk through it.